documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -110,12 +110,30 @@ Let's see the options for the `index_schema_management_strategy` property:
 [options="header"]
 |===============
 |Value|Definition
-|`NONE`|Indexes will not be created nor deleted.
+|`NONE`|Indexes and their mappings will not be created, deleted nor altered.
+|`VALIDATE`|Existing indexes and mappings will be checked for conflicts with Hibernate Search's metamodel. Indexes and their mappings will not be created, deleted nor altered.
 |`MERGE`|Missing indexes and mappings will be created, mappings will be updated if there are no conflicts.
 |`CREATE`|**The default**: existing indexes will not be altered, missing indexes will be created.
 |`RECREATE`|Indexed will be deleted if existing and then created. This will delete all content from the index!
 |`RECREATE_DELETE`|Similarly to `RECREATE` but will also delete the index at shutdown. Commonly used for tests.
 |===============
++
+[CAUTION]
+.Strategies in production environments
+====
+It is strongly recommended to use either `NONE` or `VALIDATE` in a production environment. `CREATE` and `CREATE_DELETE` are obviously unsuitable in this context (unless you want to reindex everything upon every startup), and `MERGE` may leave your mapping half-merged in case of conflict.
+
+To be precise, if your mapping changed in an incompatible way, such as a field having its type changed, merging may be impossible. In this case, the `MERGE` strategy will prevent Hibernate Search from starting, but it may already have successfully merged another index, making a rollback difficult at best.
+
+For this reason, migrating your mapping should be considered a part of your deployment process and be planned cautiously.
+====
++
+[NOTE]
+--
+Mapping validation is as permissive as possible. Fields or mappings that are unknown to Hibernate Search will be ignored, and settings that are more powerful than required (e.g. a field annotated with `@Field(store = Store.NO)` in Search but marked as `"store": true` in Elasticsearch) will be deemed valid.
```

Comment:
Given the Store.NO example (good one!), should we clarify what will happen in that case? I haven't tested it, but I'd assume that the storage operation would happen (as ES would still do it), but we wouldn't allow a projection on this field.